src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -279,7 +279,10 @@ describe('<Nav.Item />', () => {
         {location: 'bar'},
       );
 
-      item.find(UnstyledLink).simulate('click');
+      item
```

Comment:
These tests broke when `React.memo` was used instead of `PureComponent`. [Simulate will use the root node](https://airbnb.io/enzyme/docs/api/ReactWrapper/simulate.html) which I'm guessing `React.memo` interferes with